### ISSUE TITLE
Binding to a specific IP endpoint

### ISSUE THREAD

**Issue by vukovinski** [eclipse-ecal/fineftp-server#2]
Currently, the server can only be configured with a port.

It would be great if we could choose a specific network interface (via IP address) to bind to.

for example:
```cpp
std::string address = "10.1.30.88";
fineftp::FtpServer server(address, port);
```

**Comment by FlorianReimold** [comment]
Unfortunatelly you deleted your fork or made it private before I was able to merge PR #3.

**Comment by FlorianReimold** [comment]
Implemented by #5
